src/modules/FioAddress/components/SelectFioAddress.js

```diff
@@ -114,8 +117,35 @@ class SelectFioAddress extends React.Component<Props, LocalState> {
         await checkRecordSendFee(fioWallet, fioAddress)
       }
     } catch (e) {
-      error = e.message
-      showError(e.message)
+      if (e.code && e.code === FIO_NO_BUNDLED_ERR_CODE) {
+        this.props.onSelect(fioAddress, fioWallet, e.message)
+        const walletFioAddresses = await fioWallet.otherMethods.getFioAddresses()
+        const selectedFioAddress = walletFioAddresses.find(item => item.name === fioAddress)
+        const answer = await Airship.show(bridge => (
+          <ButtonsModal
+            bridge={bridge}
+            title={s.strings.fio_no_bundled_err_msg}
+            message={s.strings.fio_no_bundled_renew_err_msg}
+            buttons={{
+              ok: { label: s.strings.title_fio_renew_address },
+              cancel: { label: s.strings.string_cancel_cap, type: 'secondary' }
+            }}
+          />
+        ))
+        if (answer === 'ok') {
+          return Actions[Constants.FIO_ADDRESS_SETTINGS]({
+            showRenew: true,
+            fioWallet,
+            fioAddressName: fioAddress,
+            expiration: intl.formatExpDate(selectedFioAddress.expiration),
+            popTo: Constants.FIO_REQUEST_LIST
```

Comment:
remove popTo, leave just pop, get expiration date from redux, compare it in getDerivedStateFromProps, call setFioAddress